src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write code here
+const ulListRef = document.querySelector('.tree');
+
+ulListRef.addEventListener('click', (e) => {
+  const item = e.target;
+  const listChildrens = [...item.children];
+  const isLastLi = listChildrens.length === 0;
```

Comment:
Also, you could use a logical operator
```suggestion
  const isLastLi = !listChildrens.length;
```